apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/claimFissRead.json

```diff
@@ -182,29 +182,29 @@
       }
     },
     {
+      "sequence": 3,
       "category": {
         "coding": [
           {
+            "system": "http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBSupportingInfoType",
```

Comment:
As we saw previously, it appears these files have been edited by hand and have inconsistent formatting. Most of the changes in these json files is fixing these inconsistencies.